### ISSUE TITLE
Day 1 Slides

### ISSUE THREAD

**Issue by sethkane** [chiefkikio/AFutureDate#92]
Digital Accessibility Legal Update: https://www.slideshare.net/lflegal/a-future-date-conference-digital-accessibility-legal-update-42120

XR Access: Improving Accessibility for Virtual, Augmented and Mixed Reality: https://drive.google.com/open?id=1nPFe608sY2NhkRjEzBRdRKnaJ_uKpRkC  

Using the Black Art of Marketing to sell Accessibility: https://www.slideshare.net/Intopia/using-the-black-art-of-marketing-to-sell-accessibility-a-future-date
